api/task_check_run.go

```diff
@@ -93,6 +95,12 @@ type TaskCheckDatabaseStatementAdvisePayload struct {
 	PolicyID int `json:"policyID,omitempty"`
 }
 
+// TaskCheckStatementTypePayload is the task check payload for SQL type.
+type TaskCheckStatementTypePayload struct {
+	Statement string  `json:"statement,omitempty"`
+	DbType    db.Type `json:"dbType,omitempty"`
+}
+
```

Comment:
I see we've defined a new type of TaskCheck payload here on the server-side. But we didn't do the same thing on the frontend-side, even if it might be not used now.